src/main/java/core/basesyntax/utils/FigureSupplier.java

```diff
@@ -35,23 +37,31 @@ public Figure getDefaultFigure() {
     }
 
     private Figure getRandomCircle() {
-        return new Circle(colorSupplier.getRandomColor(), side);
+        return new Circle(colorSupplier.getRandomColor(), randomNumber(MAX_NUMBER_FOR_SIDES));
     }
 
     private Figure getRandomIsoscelesTrapezoid() {
         return new IsoscelesTrapezoid(colorSupplier
-                .getRandomColor(), side, side + 2, side);
+                .getRandomColor(), randomNumber(MAX_NUMBER_FOR_SIDES),
+                randomNumber(MAX_NUMBER_FOR_SIDES) + 2, randomNumber(MAX_NUMBER_FOR_SIDES));
     }
 
     private Figure getRandomRectangle() {
-        return new Rectangle(colorSupplier.getRandomColor(), side, side);
+        return new Rectangle(colorSupplier.getRandomColor(), randomNumber(MAX_NUMBER_FOR_SIDES),
+                randomNumber(MAX_NUMBER_FOR_SIDES));
     }
 
     private Figure getRandomRightTriangle() {
-        return new RightTriangle(colorSupplier.getRandomColor(), side, side);
+        return new RightTriangle(colorSupplier.getRandomColor(), randomNumber(MAX_NUMBER_FOR_SIDES),
+                randomNumber(MAX_NUMBER_FOR_SIDES));
     }
 
     private Figure getRandomSquare() {
-        return new Square(colorSupplier.getRandomColor(), side);
+        return new Square(colorSupplier.getRandomColor(), randomNumber(MAX_NUMBER_FOR_SIDES));
+    }
+
+    private int randomNumber(int number) {
+        Random random = new Random();
```

Comment:
Let's move this line to a class level, because otherwise a new object of type Random will be created each time this method is called